ProjectHeads/App.Head.props

```diff
@@ -76,6 +76,48 @@
       </ItemGroup>
     </When>
   </Choose>
+
+  <!-- Gather sample markdown for components that MultiTarget this platform. -->
+  <!-- Each deployable sample app head uses the CommunityToolkit.Toolkit.SampleGen source generator to generate document registries -->
+  <Import Project="$(ToolingDirectory)\ProjectHeads\Tasks\CheckMultiTarget.props" />
+
+  <!-- Simplifies comparison of the current head against component-level <MultiTarget> definition -->
```

Comment:
This file is getting pretty big now, should we either move this into its own props file as well (just from an organizatinal perspective)? (I do like just having the task itself in a single file.)